### PR TITLE
CMOS-459 Release note generation

### PR DESCRIPTION
**Pull request by aemery-cb**
Generates release notes automatically based off Jira releases 🎈

### CODE REVIEW

File: tools/release_notes.go [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+func (jc *JiraClient) GetIssues(projectName, fixVersion string) ([]Issue, error) {
+
+	reqJson, err := json.Marshal(map[string]any{
+		"jql":    "fixVersion = " + fixVersion + " AND project=" + projectName + " AND resolution != Unresolved",
+		"fields": []string{"resolution", "status", "summary", "description", "issuetype"}})
+
+	if err != nil {
+		return nil, err
+	}
+	responseBody := bytes.NewBuffer(reqJson)
```
Comment (malscent) [inline tools/release_notes.go:72]: You're storing the requestBody in the "responseBody" variable?

File: tools/release_notes.go [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+	res, err := jc.client.Do(request)
+	if err != nil {
+		return nil, err
+	}
+
+	defer res.Body.Close()
+
+	queryResponse, err := ioutil.ReadAll(res.Body)
+	if 200 > res.StatusCode && res.StatusCode > 300 {
```
Comment (malscent) [inline tools/release_notes.go:88]: Won't 300 be included here?

**Review by malscent — CHANGES_REQUESTED** [review]

File: tools/release_notes.go (hunk 2 above)
Comment (aemery-cb) [reply tools/release_notes.go]: yes it will, fixed.

File: tools/release_notes.go (hunk 1 above)
Comment (aemery-cb) [reply tools/release_notes.go]: ... yes. (fixed)

**Review by malscent — APPROVED** [review]

**Pull request merged**